src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1460,17 +1460,28 @@ private char Backtick(char c, out char surrogateCharacter)
 
```

Comment:
@vexx32, your last commit had 6 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs).Numerical Notations Tests (starting at line 2374 to line 2452).0x80000000 should return -2147483648, with type [int]
```powershell
Expected -2147483648, but got 2147483648.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1: line 1143
1143:             ExecuteCommand $Script | Should -Be $ExpectedValue
```
ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs).Numerical Notations Tests (starting at line 2374 to line 2452).0xFFFFFFFF should return -1, with type [int]
```powershell
Expected '-1', but got 4294967295.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1: line 1143
1143:             ExecuteCommand $Script | Should -Be $ExpectedValue
```
ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs).Numerical Notations Tests (starting at line 2374 to line 2452).0xFFFFFFFFFFFFFFFF should return -1, with type [long]
```powershell
Exception calling "Invoke" with "0" argument(s): "At line:1 char:19
+ 0xFFFFFFFFFFFFFFFF
+                   ~
The numeric constant 0xFFFFFFFFFFFFFFFF is not valid."
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1: line 81
81:                 $PowerShell.AddScript($command).Invoke()

```
ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs).Numerical Notations Tests (starting at line 2374 to line 2452).0xFFy should return -1, with type [sbyte]
```powershell
Exception calling "Invoke" with "0" argument(s): "At line:1 char:6
+ 0xFFy
+      ~
The numeric constant 0xFFy is not valid."
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1: line 81
81:                 $PowerShell.AddScript($command).Invoke()

```
ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs).Numerical Notations Tests (starting at line 2374 to line 2452).0xFFFFs should return -1, with type [short]
```powershell
Exception calling "Invoke" with "0" argument(s): "At line:1 char:8
+ 0xFFFFs
+        ~
The numeric constant 0xFFFFs is not valid."
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1: line 81
81:                 $PowerShell.AddScript($command).Invoke()

```